### ISSUE TITLE
allow exclusive bounds

### ISSUE THREAD

**Issue by mb706** [berndbischl/ParamHelpers#168]
some learners in mlr allow for ranges of numeric parameters with exclusive bounds. This is currently worked around using `.Machine$double.eps` (and fails if the absolute value of the bound is >= 2), and should have a proper solution.
I suggest parameters (and defaults) `inclusive.bounds = TRUE`, `inclusive.bound.lower = inclusive.bounds`, `inclusive.bound.upper = inclusive.bounds`.

**Comment by berndbischl** [comment]
yeah, i thought about this some time ago, then kinda was too lazy to do this. 

i guess we could add this
